index.js

```diff
@@ -96,12 +96,21 @@ function jade_classes(val, escaping) {
 
 exports.style = jade_style;
 function jade_style(val) {
-  if (val && typeof val === 'object') {
-    return Object.keys(val).map(function (style) {
-      return style + ':' + val[style];
-    }).join(';');
+  if (!val) return '';
+  if (typeof val === 'object') {
+    var out = '', delim = '';
+    for (var style in val) {
+      /* istanbul ignore else */
+      if (val.hasOwnProperty(style)) {
+        out = out + delim + style + ':' + val[style];
+        delim = ';';
+      }
+    }
+    return out;
   } else {
-    return ('' + (val || '')).replace(/\;$/, '');
+    val = '' + val;
```

Comment:
To be honest, I don't know… I'm just trying to prevent any unintended changes in a non-major bump.